libhb/qsv_common.c

```diff
@@ -2265,6 +2279,550 @@ void hb_qsv_force_workarounds()
 #undef FORCE_WORKAROUNDS
 }
 
+#include "hb.h"
```

Comment:
Does this file not already include this above?

Reply:
nope